flink-kubernetes-operator/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,113 @@
+- commons-cli:commons-cli:1.3.1
+- commons-collections:commons-collections:3.2.2
+- commons-io:commons-io:2.8.0
+- io.fabric8:kubernetes-client:5.12.1
+- io.fabric8:kubernetes-model-admissionregistration:5.12.1
+- io.fabric8:kubernetes-model-apiextensions:5.12.1
+- io.fabric8:kubernetes-model-apps:5.12.1
+- io.fabric8:kubernetes-model-autoscaling:5.12.1
+- io.fabric8:kubernetes-model-batch:5.12.1
+- io.fabric8:kubernetes-model-certificates:5.12.1
+- io.fabric8:kubernetes-model-common:5.12.1
+- io.fabric8:kubernetes-model-coordination:5.12.1
+- io.fabric8:kubernetes-model-core:5.12.1
+- io.fabric8:kubernetes-model-discovery:5.12.1
+- io.fabric8:kubernetes-model-events:5.12.1
+- io.fabric8:kubernetes-model-extensions:5.12.1
+- io.fabric8:kubernetes-model-flowcontrol:5.12.1
+- io.fabric8:kubernetes-model-metrics:5.12.1
+- io.fabric8:kubernetes-model-networking:5.12.1
+- io.fabric8:kubernetes-model-node:5.12.1
+- io.fabric8:kubernetes-model-policy:5.12.1
+- io.fabric8:kubernetes-model-rbac:5.12.1
+- io.fabric8:kubernetes-model-scheduling:5.12.1
+- io.fabric8:kubernetes-model-storageclass:5.12.1
+- io.fabric8:openshift-client:5.12.0
+- io.fabric8:openshift-model-clusterautoscaling:5.12.0
+- io.fabric8:openshift-model-console:5.12.0
+- io.fabric8:openshift-model-hive:5.12.0
+- io.fabric8:openshift-model-installer:5.12.0
+- io.fabric8:openshift-model-machine:5.12.0
+- io.fabric8:openshift-model-machineconfig:5.12.0
+- io.fabric8:openshift-model-miscellaneous:5.12.0
+- io.fabric8:openshift-model-monitoring:5.12.0
+- io.fabric8:openshift-model-operator:5.12.0
+- io.fabric8:openshift-model-operatorhub:5.12.0
+- io.fabric8:openshift-model-storageversionmigrator:5.12.0
+- io.fabric8:openshift-model-tuned:5.12.0
+- io.fabric8:openshift-model-whereabouts:5.12.0
+- io.fabric8:openshift-model:5.12.0
+- io.fabric8:zjsonpatch:0.3.0
+- io.javaoperatorsdk:operator-framework-core:2.1.2
+- io.javaoperatorsdk:operator-framework:2.1.2
+- org.apache.commons:commons-compress:1.21
+- org.apache.commons:commons-lang3:3.12.0
+- org.apache.commons:commons-math3:3.5
+- org.apache.flink:flink-annotations:1.14.4
+- org.apache.flink:flink-clients_2.12:1.14.4
+- org.apache.flink:flink-core:1.14.4
+- org.apache.flink:flink-file-sink-common:1.14.4
+- org.apache.flink:flink-hadoop-fs:1.14.4
+- org.apache.flink:flink-java:1.14.4
+- org.apache.flink:flink-metrics-core:1.14.4
+- org.apache.flink:flink-optimizer:1.14.4
+- org.apache.flink:flink-queryable-state-client-java:1.14.4
+- org.apache.flink:flink-rpc-akka-loader:1.14.4
+- org.apache.flink:flink-rpc-core:1.14.4
+- org.apache.flink:flink-runtime:1.14.4
+- org.apache.flink:flink-shaded-asm-7:7.1-14.0
+- org.apache.flink:flink-shaded-force-shading:14.0
+- org.apache.flink:flink-shaded-guava:30.1.1-jre-14.0
+- org.apache.flink:flink-shaded-jackson:2.12.4-14.0
+- org.apache.flink:flink-shaded-netty:4.1.65.Final-14.0
+- org.apache.flink:flink-shaded-zookeeper-3:3.4.14-14.0
+- org.apache.flink:flink-streaming-java_2.12:1.14.4
```

Comment:
Do we need to list Flink dependencies here?

Reply:
Since `flink-kubernetes-operator` is a separate project and we bundle Flink jars, that's why I also list the Flink dependencies here. Maybe you are right. But I am not sure.

Reply:
we can leave it there for now, no harm here. But we can revisit this later and remove if not necessary

Reply:
All Flink artifacts come with a nice notice file that if properly merged should remove the need to explicitly mention it here.